resources/locale.json

```diff
@@ -745,195 +745,195 @@
-        "UI_PEACOCK_HEDGEBUSH_GOLD_DESC": "Steal all the gold from the vault.",
-        "UI_PEACOCK_HEDGEBUSH_BURGERS": "Steal the Burgers",
-        "UI_PEACOCK_HEDGEBUSH_BURGERS_DESC": "Steal all the burgers from the bank.",
-        "UI_PEACOCK_HEDGEBUSH_OBJ1": "Eliminate Frank Winkelman with a Cheeseburger as the Bank Robber",
-        "UI_PEACOCK_HEDGEBUSH_OBJ2": "Eliminate Gourd with a Cheeseburger",
+        "UI_PEACOCK_HEDGEBUSH_GOLD": "Stehle das Gold",
+        "UI_PEACOCK_HEDGEBUSH_GOLD_DESC": "Stehle das komplette Gold aus dem Tresor.",
+        "UI_PEACOCK_HEDGEBUSH_BURGERS": "Stehle die Burger",
+        "UI_PEACOCK_HEDGEBUSH_BURGERS_DESC": "Stehle alle Burger aus der Bank.",
+        "UI_PEACOCK_HEDGEBUSH_OBJ1": "Eliminiere Frank Winkelman mit einem Cheeseburger als Bank Räuber verkleidet",
+        "UI_PEACOCK_HEDGEBUSH_OBJ2": "Eliminiere Gourd mit einem Cheeseburger",
         "UI_PEACOCK_HEDGEBUSH_GOLDREMAINING": "Gold verbleibend",
         "UI_PEACOCK_HEDGEBUSH_BURGERSREMAINING": "Burger verbleibend",
         "UI_PEACOCK_BLUEBERRYBUSH": "Die PurpleKey Gefahr",
         "UI_PEACOCK_BLUEBERRYBUSH1": "Die PurpleKey Gefahr - Stufe 1",
         "UI_PEACOCK_BLUEBERRYBUSH2": "Die PurpleKey Gefahr - Stufe 2",
         "UI_PEACOCK_BLUEBERRYBUSH3": "Die PurpleKey Gefahr - Stufe 3",
-        "UI_PEACOCK_BLUEBERRYBUSH_OBJ1": "Eliminate Claudia Di Meo in an accident",
-        "UI_PEACOCK_BLUEBERRYBUSH_OBJ2": "Eliminate Edoardo Pugliesi in an accident",
-        "UI_PEACOCK_BLUEBERRYBUSH_OBJ3": "Eliminate Aldo Trentino in an accident",
+        "UI_PEACOCK_BLUEBERRYBUSH_OBJ1": "Eliminiere claudia Di Meo durch einen Unfall",
+        "UI_PEACOCK_BLUEBERRYBUSH_OBJ2": "Eliminiere Edoardo Pugliesi durch einen Unfall",
+        "UI_PEACOCK_BLUEBERRYBUSH_OBJ3": "Eliminiere Aldo Trentino durch einen Unfall",
         "UI_PEACOCK_GLOBAL_CAMERA_FAIL": "Kamerascheu",
         "UI_PEACOCK_GLOBAL_CAMERA_FAIL_DESC": "Du scheiterst wenn du von einer Kamera entdeckt wirst.",
         "UI_PEACOCK_GRAPEBUSH": "Die Argentine Bitterkeit",
         "UI_PEACOCK_GRAPEBUSH1": "Die Argentine Bitterkeit - Stufe 1",
         "UI_PEACOCK_GRAPEBUSH2": "Die Argentine Bitterkeit - Stufe 2",
         "UI_PEACOCK_GRAPEBUSH3": "Die Argentine Bitterkeit - Stufe 3",
-        "UI_PEACOCK_GRAPEBUSH_FILES": "Get the Leak Report from the Basement Safe",
-        "UI_PEACOCK_GRAPEBUSH_FILES_DESC": "Get the Leak Report from the Basement Safe.",
-        "UI_PEACOCK_GRAPEBUSH_OBJ1": "Eliminate Violetta Mendez in an accident",
-        "UI_PEACOCK_GRAPEBUSH_OBJ2": "Eliminate Montel Beneventi in an explosion or with a cannon",
-        "UI_PEACOCK_GRAPEBUSH_OBJ2_DESC": "Eliminate Montel Beneventi in an explosion or with a cannon.",
-        "UI_PEACOCK_GRAPEBUSH_OBJ3": "[Optional] Eliminate Hira Amala",
-        "UI_PEACOCK_GRAPEBUSH_OBJ3_DESC": "If you're up for it, eliminate Hira Amala.",
+        "UI_PEACOCK_GRAPEBUSH_FILES": "Besorge den Leckbericht aus dem Kellertresor",
```

Comment:
"Leckbericht" sounds weird, the first thing I think of when I hear that is licking lmao